### PR TITLE
Feat/set monoruntime as default kre

### PR DESCRIPTION
**Pull request by IGZangelcostales**

### CODE REVIEW

File: helm/kre/templates/config/configmap.yaml [hunk 1]
```diff
@@ -5,9 +5,12 @@ metadata:
   name: {{ .Release.Name }}-configmap
   labels:
     {{- include "kre.labels" . | nindent 4 }}
-    app: {{ template "kre.fullname" . }}-admin-api
+    app: {{ .Release.Name }}-admin-api
 data:
   KRE_DEVELOPMENT_MODE: "{{ .Values.developmentMode }}"
+  KRE_MONORUNTIME_MODE: "yes"
```
Comment (guumaster) [inline helm/kre/templates/config/configmap.yaml:11]: maybe this variable can be removed on this refactor.

File: helm/kre/.helmignore [hunk 2]
```diff
@@ -1,23 +0,0 @@
-# Patterns to ignore when building packages.
```
Comment (guumaster) [inline helm/kre/.helmignore:1]: maybe this file should be moved instead of removed?

File: helm/kre/templates/config/configmap.yaml (hunk 1 above)
Comment (guumaster) [reply helm/kre/templates/config/configmap.yaml]: removed

File: engine/admin-api/domain/usecase/runtime.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	if err != nil {
+		i.logger.Errorf("Error creating runtime: %s", err)
+		return err
+	}
+	i.logger.Info("Runtime stored in the database with ID=" + createdRuntime.ID)
+
+	return nil
+}
+
+// GetByID return a Runtime by its ID
```
Comment (guumaster) [inline engine/admin-api/domain/usecase/runtime.go:82]: Fix comment to `Get`

**Review by danielchg — APPROVED** [review]

**Pull request merged**